pkgs/servers/pulseaudio/qpaeq.nix

```diff
@@ -0,0 +1,66 @@
+{ mkDerivation
+, python3
+, fetchurl
+, lib
+, pulseaudio
+}:
+
+let
+  qp = "./src/utils/qpaeq";
+in
+mkDerivation rec {
+  pname = "pulseuadio-qpaeq-${version}";
+  version = "13.0";
+
+  /* src = fetchurl {
+    url = "http://freedesktop.org/software/pulseaudio/releases/pulseaudio-${version}.tar.xz";
+    sha256 = "0mw0ybrqj7hvf8lqs5gjzip464hfnixw453lr0mqzlng3b5266wn";
+  }; */
```

Comment:
why is this commented?

Reply:
Unsure how we should best make it so pulse & qpaeq always reference the same url

Reply:
You got any idea?

Reply:
I guess there's a way to reuse the `src` of the original pulseaudio derivation... Not sure though...

Reply:
`pulseaudio.src` should work.

Reply:
You would then set `${pulseaudio.src.name}/src/utils` or something.

Reply:
Added it to src